Gemfile

```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
 
+gem 'nsq-ruby', '~> 1.5.0', '>= 1.5.0', git: 'https://github.com/fastly/nsq-ruby.git'
+
```

Comment:
This seems like a fine change:

> If you find that you need to develop against a gem that hasn't yet been released (for instance, Rails often develops against unreleased Rack, Arel, or Mail gems), you can add individual lines in the Gemfile that tell bundler where to find the gems. It will still use the dependencies listed in the .gemspec for dependency resolution, but it now knows exactly where to find the dependency during gem development.

http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/